product-mini/platforms/linux-sgx/enclave-sample/Enclave/Enclave.cpp

```diff
@@ -382,14 +385,15 @@ handle_cmd_set_wasi_args(uint64 *args, int32 argc)
         p += sizeof(char *) * wasi_argc;
     }
 
-    wasm_runtime_set_wasi_args(enclave_module->module,
-                               (const char **)enclave_module->wasi_dir_list,
-                               dir_list_size,
-                               NULL, 0,
-                               (const char **)enclave_module->wasi_env_list,
-                               env_list_size,
-                               enclave_module->wasi_argv,
-                               enclave_module->wasi_argc);
+    wasm_runtime_set_wasi_args_ex(enclave_module->module,
+                                  (const char **)enclave_module->wasi_dir_list,
+                                  dir_list_size,
+                                  NULL, 0,
+                                  (const char **)enclave_module->wasi_env_list,
+                                  env_list_size,
+                                  enclave_module->wasi_argv,
+                                  enclave_module->wasi_argc,
+                                  stdinfd, stdoutfd, stderrfd);
```

Comment:
The input fd may be -1, should convert them here?
```C
stdinfd == -1 ? stdinfd : 0,
stdoutfd == -1 ? stdoutfd : 1,
stderrfd == -1 ? stderrfd : 0
```

Reply:
fixed